vite.config.ts

```diff
@@ -3,7 +3,6 @@ import { sveltekit } from "@sveltejs/kit/vite";
 import { partytownVite } from "@builder.io/partytown/utils";
 import svg from "@poppanator/sveltekit-svg";
 import type { PluginOption } from "vite";
-import { purgeCss } from "vite-plugin-svelte-purgecss";
```

Comment:
@LouisFettet @hinzed1127 What happened when I upgraded the packages and corresponding dependencies was that `purgeCss` has changed to only support tailwindCSS. Should I try to replace this package?

Reply:
that package was still in flux the last time I tried doing that, but it's probably worth another look. it looks like [the new plugin has a legacy mode](https://github.com/AdrianGonz97/vite-plugin-tailwind-purgecss/blob/master/legacy-mode.md) that targets beyond tailwind classes, with the caveat that it had more bugs. It's worth bringing it in and seeing if it hopefully still works the same way. If not, we can try to troubleshoot further

Reply:
@hinzed1127 Thanks for telling me about this. It looks like it works, but I have to put in an empty tailwind.config.js file 🤷